internal/catalog/internal/controllers/failover/status.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package failover
+
+import (
+	"github.com/hashicorp/consul/internal/resource"
+	"github.com/hashicorp/consul/proto-public/pbresource"
+)
+
+const (
+	StatusKey              = "consul.io/failover-policy"
+	StatusConditionHealthy = "healthy"
```

Comment:
Is this used?

Reply:
That's the condition type for all of the conditions.